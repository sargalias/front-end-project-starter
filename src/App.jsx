import React from 'react';
import styles from './App.scss';

const App = () => (
  <div className={styles.App}>
    <h1 className={styles.App_heading}>App</h1>
    <p className={styles.App_text}>Text</p>
  </div>
);

export default App;
