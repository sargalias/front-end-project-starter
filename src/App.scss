.App {
  max-width: 1000px;
  padding: 1rem;
  border: 1px solid $color-sample;
  margin: 0 auto;

  &_heading {
    text-align: center;
  }
}
