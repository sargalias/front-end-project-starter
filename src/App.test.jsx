import React from 'react';
import { render } from 'testUtils';
import App from './App';

test('snapshot', () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
