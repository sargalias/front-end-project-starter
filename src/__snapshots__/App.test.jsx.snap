// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div>
  <div
    class="App"
  >
    <h1
      class="App_heading"
    >
      App
    </h1>
    <p
      class="App_text"
    >
      Text
    </p>
  </div>
</div>
`;
